deps/wazuh_testing/wazuh_testing/tools/file.py

```diff
@@ -51,6 +51,22 @@ def read_yaml(file_path):
         return yaml.safe_load(f)
 
 
+def get_list_of_content_yml(file_path, filter, list=[]):
+    """Read a YAML file from a given path, return a list with the YAML data
+    after apply filter
+    
+    Args:
+        file_path (str): Path of the YAML file to be readed
+        filter (str): filder to extract some part of yaml
+        list: a list with previous information
+    
+    Returns:
+       list: Yaml structure.
+    """
+    list.append((read_yaml(file_path), file_path + filter))
+    return list
+
+
```

Comment:
If we don't use this `method `but this block is repeat on each test, 

> with open(messages_file) as f:
>     module_tests.append((yaml.safe_load(f), messages_file.split('_')[0]))
> 
Could we add this content on a this method an use it on each test?